core/optaplanner-core-impl/src/main/java/org/optaplanner/core/config/heuristic/selector/move/generic/list/kopt/KOptListMoveSelectorConfig.java

```diff
@@ -0,0 +1,61 @@
+package org.optaplanner.core.config.heuristic.selector.move.generic.list.kopt;
+
+import java.util.function.Consumer;
+
+import javax.xml.bind.annotation.XmlType;
+
+import org.optaplanner.core.config.heuristic.selector.move.MoveSelectorConfig;
+import org.optaplanner.core.config.util.ConfigUtils;
+
+@XmlType(propOrder = {
+        "minimumK",
+        "maximumK",
+})
+public class KOptListMoveSelectorConfig extends MoveSelectorConfig<KOptListMoveSelectorConfig> {
+
+    public static final String XML_ELEMENT_NAME = "kOptListMoveSelector";
+
+    protected Integer minimumK = null;
+    protected Integer maximumK = null;
```

Comment:
Please add `.with...()` methods for these, so that people can easily build fluent configs in Java.